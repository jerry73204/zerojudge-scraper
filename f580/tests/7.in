10 59
8 -2
9 3
6 5
3 8
7 9
9 -2
3 5
8 -1
5 -1
5 2
5 -1
2 -2
6 -2
7 2
3 8
5 1
2 7
10 7
10 -2
5 2
9 8
1 10
2 -1
9 -1
7 10
10 4
10 -2
5 6
7 -2
7 -1
10 8
4 6
5 3
1 -2
2 8
10 7
1 4
4 3
8 2
2 1
10 1
2 8
7 2
5 2
10 3
9 8
3 1
6 10
9 -2
8 -1
3 4
3 2
1 2
5 8
3 4
5 7
7 -2
5 9
4 5
