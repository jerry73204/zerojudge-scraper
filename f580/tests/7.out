5 3 6 2 3 6 2 5 5 1